cmd/neofs-lens/internal/meta/get.go

```diff
@@ -16,28 +16,35 @@ var getCMD = &cobra.Command{
 	Short: "Object inspection",
 	Long:  `Get specific object from a metabase.`,
 	Args:  cobra.NoArgs,
-	Run:   getFunc,
+	RunE:  getFunc,
 }
 
 func init() {
 	common.AddAddressFlag(getCMD, &vAddress)
 	common.AddComponentPathFlag(getCMD, &vPath)
 }
 
-func getFunc(cmd *cobra.Command, _ []string) {
+func getFunc(cmd *cobra.Command, _ []string) error {
 	var addr oid.Address
 
 	err := addr.DecodeString(vAddress)
-	common.ExitOnErr(cmd, common.Errf("invalid address argument: %w", err))
+	if err != nil {
+		return common.Errf("invalid address argument: %w", err)
```

Comment:
`common.Errf` made sense with `ExitOnErr` func since it allowed to avoid `if` statement. Now we should use `fmt.Errorf` (and purge `Errf`)